src/Application/inc/CalculatorApplication.h

```diff
@@ -39,7 +39,7 @@ class CalculatorApplication
          * @param [in] operation operator char
          */
 
-        static int calculate(float firstNumber, float secondNumber, char operation, std::string originalEquation);
+        static int calculate(float firstNumber, float secondNumber, char operation, std::string const &originalEquation);
```

Comment:
the const should be before the type. Apparently it means the same thing if it is there... However it means something different if it is `std::string& const originalEquation` and it could be easy to confuse the two

Anyway, it should look like this:

`const std::string &originalEquation` 

https://stackoverflow.com/questions/5503352/const-before-or-const-after